File: NodeJS/debug/debug_console.js
const { chromium } = require('playwright');

(async () => {
  const browser = await chromium.launch();
  const page = await browser.newPage();

  page.on('console', msg => {
    console.log(`Console ${msg.type()}: ${msg.text()}`);
  });

  await page.goto('https://example.com');
  await page.evaluate(() => console.log('Hello from browser console'));
  await browser.close();
})();

File: Python/debug/debug_console.py
from playwright.sync_api import sync_playwright

def main():
    with sync_playwright() as p:
        browser = p.chromium.launch()
        page = browser.new_page()
        page.on("console", lambda msg: print(f"Console: {msg.type}: {msg.text}"))
        page.goto("https://example.com")
        page.evaluate("console.log('Hello from browser')")
        browser.close()

main()
